tests/test_io.py

```diff
@@ -68,6 +69,18 @@ def test_imaging_extractors(self, extractor_class, extractor_kwargs):
         except NotImplementedError:
             return
 
+    @parameterized.expand(imaging_extractor_list, name_func=custom_name_func)
+    def test_imaging_extractors_canonical_shape(self, extractor_class, extractor_kwargs):
+        extractor = extractor_class(**extractor_kwargs)
+        image_size = extractor.get_image_size()
+        num_channels = extractor.get_num_channels()
+        video = extractor.get_video()
+
+        canonical_video_shape = [extractor.get_num_frames(), image_size[0], image_size[1]]
```

Comment:
So, if image_size[0] is columns and image_size[0] is rows this will work but it should not. Also, no need to load the whole video to test this.

Reply:
Oh, I realized this is merged already.